### ISSUE TITLE
Refresh button does nothing

### ISSUE THREAD

**Issue by rafnov** [Droid-ify/client#750]
On my Realme 7 pro the refresh button does not what it is supposed to: does not refresh repos. Just checked on another smartphone and it refreshed.
How you fix it?

**Comment by rafnov** [comment]
Anyone?

**Comment by Iamlooker** [comment]
If it is latest repo than it will not refresh

**Comment by rafnov** [comment]
It does not show any progress on my one particular device: Realme 7 pro. Even if the repos are latest I would expect some message about that, but no such thing.

**Comment by rafnov** [comment]
I can confirm on another devices: Droid-ify shows or does not show any notification depending on the device.
1. Realme 7 Pro (Android 12) and Realme GT Neo 2 (Android 13) - no progress of updating repos or any message at all (if repos would be up to date)
2. There is notification of progress updating repos on another devices like Samsung Galaxy S5 (LineageOS / Android 11) or Lenovo Yoga Smart Tab (Android 10).

**Comment by Iamlooker** [comment]
Is notification permission allowed to Droid-ify? Android 12+ need to allow that permission to show notification

**Comment by rafnov** [comment]
Yes, notification permissions are enabled on both Realme devices.

**Comment by Iamlooker** [comment]
Weird, I will need to test it

**Comment by Iamlooker** [comment]
Now shows a progress indicator when syncing

Implemented with 8435c8f7eada3c2b0b12b9b40dffee3557ba517f
